nightguard.xcodeproj/project.pbxproj

```diff
@@ -2594,7 +2598,7 @@
 				ENABLE_BITCODE = NO;
 				EXCLUDED_ARCHS = "";
 				INFOPLIST_FILE = nightguard/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 14.7;
+				IPHONEOS_DEPLOYMENT_TARGET = 16.0;
```

Comment:
Was this by accident? Or is iOS 16.0 required?

Reply:
I saw that version 16 was already set in the "nightguard Widget Extension" and I needed at least version 15 for ISO8601FormatStyle, so I harmonized everything to 16